Automattic/vip
# Allows user to continue even when the answer doesn't match the expected letter casing

## Description

The vip cli will ask for users to type the environment url before continuing. But it would only accept exact matches and would exit with an error if the answer was correct, but with a different letter casing. This PR fixes that.

## Pull request checklist


## New release checklist


## Steps to Test

1. Check out PR.
1. Run `npm run build`
1. Run `./dist/bin/vip-import-sql.js -a 3800 test.sql -s`
1. When asked to type the environment url, answer with lower case
1. Verify that the import starts
